test/behavior/tendrl/cluster.feature

```diff
@@ -0,0 +1,98 @@
+Feature: CLUSTER
+
+  Scenario: List view is available
+    Given I open the Tendrl url
+    Then  I should see the "cluster" data
+
+  Scenario: Import the Clusters 
+    Given I open the Tendrl url
+    When  I should see the "import" button
+    And   I click on "import" button
+    Then  I should see the "import" "cluster" page
+
+  Scenario: Create the Clusters 
+    Given I open the Tendrl url
+    When  I should see the "create" button
+    And   I click on Create button
+    Then  I should see the "add" "cluster" page
+
+  Scenario: Search the Clusters by Name
+    Given I open the Tendrl url
+    When  I type the "name" of the Cluster available
+    Then  I should see the Cluster in List View
```

Comment:
@gnehapk yes, ideally we should do it. That is in my bucket list to add the negative test once all the base scenarios are covered.